src/Core/Grid/Definition/Factory/Monitoring/AbstractProductGridDefinitionFactory.php

```diff
@@ -187,16 +191,20 @@ protected function getRowActions()
                     ])
             )
             ->add(
-                (new LinkRowAction('delete'))
-                    ->setName($this->trans('Delete', [], 'Admin.Actions'))
-                    ->setIcon('delete')
-                    ->setOptions([
-                        'route' => 'admin_product_unit_action',
-                        'route_param_name' => 'id',
-                        'route_param_field' => 'id_product',
-                        'extra_route_params' => ['action' => 'delete'],
-                        'confirm_message' => $this->trans('Delete selected item?', [], 'Admin.Notifications.Warning'),
-                    ])
+                $this->buildDeleteAction(
+                    'admin_product_unit_action',
+                    'id',
+                    'id_product',
+                    Request::METHOD_DELETE,
+                    ['action' => 'delete'],
+                    [
+                        'confirm_message' => $this->trans(
+                            'Delete selected item?',
```

Comment:
Can you confirm this message is displayed only when a single item is selected?

Reply:
Yes it's only for single deletion